examples/sdk-app-example/src/main/AndroidManifest.xml

```diff
@@ -35,11 +35,11 @@
       android:value="YOUR-API-KEY-HERE" />
 
     <meta-data
-      android:name="com.bugsnag.android.DETECT_ANRS"
+      android:name="com.bugsnag.android.AUTO_DETECT_ANRS"
       android:value="true" />
 
     <meta-data
-      android:name="com.bugsnag.android.DETECT_NDK_CRASHES"
+      android:name="com.bugsnag.android.AUTO_DETECT_NDK_CRASHES"
```

Comment:
Assuming the default value for this configuration option is true, it is potentially confusing for it to be in the manifest like this. Maybe having it commented out like this would be more clear so that people don't think they have to include it:

```xml
<!-- NDK crash detection is enabled by default. To disable it, set the following option: -->
<!--
  <meta-data
    android:name="com.bugsnag.android.AUTO_DETECT_NDK_CRASHES"
    android:value="false" />
-->
```

Reply:
NDK crash detection is currently disabled by default so we need to include this in the manifest for the example app.